src/zenml/services/local/local_daemon_entrypoint.py

```diff
@@ -22,6 +22,10 @@
 
 from zenml.utils.daemon import daemonize
 
+# import this explicitly for the service daemon to know about this
```

Comment:
Can you explain who needs to now about this/where we need to explicitly import this file?